build.gradle

```diff
@@ -66,13 +66,13 @@ wrapper {
 ext {
     moduleName = "io.aiven.kafka.connect.http"
 
-    avroVersion = "1.8.1" // Version 1.8.1 brings Jackson 1.9.x/org.codehaus.jackson package for Avro and Confluent Platform 4.1.4.
-    confluentPlatformVersion = "4.1.4" // For compatibility tests use version 4.1.4.
+    avroVersion = "1.10.0" // Version 1.8.1 brings Jackson 1.9.x/org.codehaus.jackson package for Avro and Confluent Platform 4.1.4.
```

Comment:
Do we really need to upgrade all these dependencies in this PR except `kafkaVersion`?

Reply:
This was part of me trying to figure out the dependency issues and I didn't think to move this one back once I had something working. I moved it back.